### PR TITLE
[release/v1.5] Add K8s v1.27.5

### PR DESCRIPTION
**Pull request by rayandas**
Related: https://github.com/rancher/rancher/issues/41840

### CODE REVIEW

File: cluster/plan.go [hunk 1]
```diff
@@ -505,7 +505,7 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, serviceOptions v3.Kubern
 	var Binds []string
 
 	if c.IsCRIDockerdEnabled() {
-		CommandArgs["container-runtime"] = "remote"
+		//CommandArgs["container-runtime"] = "remote" https://v1-26.docs.kubernetes.io/docs/reference/command-line-tools-reference/kubelet/
```
Comment (kinarashah) [inline cluster/plan.go]: Does `container-runtime` default to `remote` for <v1.27 versions? If not, we should add a separate check for >=v1.27. Need to make sure we don't break older versions by removing this field.

File: docker/docker.go [hunk 2]
```diff
@@ -400,11 +400,11 @@ func StopContainer(ctx context.Context, dClient *client.Client, hostname string,
 	}
 	var err error
 	// define the stop timeout
-	stopTimeoutDuration := StopTimeout * time.Second
+	stopTimeoutDuration := StopTimeout
```
Comment (kinarashah) [inline docker/docker.go]: don't need this anymore as `StopTimeout` can be used directly, same for `RestartTimeout` above.

File: cluster/plan.go (hunk 1 above)
Comment (rayandas) [reply cluster/plan.go]: No I don't think it defaults to remote for <v1.27 versions. Will add a separate check here.

File: docker/docker.go (hunk 2 above)
Comment (rayandas) [reply docker/docker.go]: Yeah right. Updating it.

**Review by kinarashah — APPROVED** [review]

**Pull request merged**